### PR TITLE
Improve ReadMe.md files

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**

### CODE REVIEW

File: LogPayload/ReadMe.md [hunk 1]
```diff
@@ -25,4 +25,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: sonarsource/gh-action-lt-backlog/LogPayload@v2
+
```
Comment (pavel-mikula-sonarsource) [inline LogPayload/ReadMe.md:28]: When copying the content, file is created without the last EOL

File: PullRequestClosed/ReadMe.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# SubmitReview
+# PullRequestClosed
```
Comment (pavel-mikula-sonarsource) [inline PullRequestClosed/ReadMe.md:1]: Oops

File: PullRequestClosed/ReadMe.md [hunk 3]
```diff
@@ -27,15 +27,15 @@ None
...
 
 on:
   pull_request:
     types: [closed]
 
 jobs:
-  MergePullRequest_job:
-    name: Submit Review
+  PullRequestMerged_job:
+    name: Pull Request Merged
```
Comment (pavel-mikula-sonarsource) [inline PullRequestClosed/ReadMe.md:38]: Oops

File: PullRequestCreated/ReadMe.md [hunk 4]
```diff
@@ -64,6 +64,8 @@ jobs:
     # For external PR, ticket should be created manually
     if: |
         github.event.pull_request.head.repo.full_name == github.repository
+    permissions:
+      id-token: write
```
Comment (pavel-mikula-sonarsource) [inline PullRequestCreated/ReadMe.md:68]: Because it uses the special GH token, like RequestReview

**Review by sebastien-marichal — APPROVED** [review]
LGTM!

**Pull request merged**